app/assets/javascripts/scheduler.js

```diff
@@ -151,7 +151,7 @@ $(document).ready(function() {
 			agendaWeek: 'yyyy'
 		},
 		// Sets height of the plugin calendar
-		contentHeight: 610,
+		contentHeight: 450,
```

Comment:
This lets you see the schedule up to around 4pm and scroll it to see more. It just makes it inline with the height of the sidebar